### PR TITLE
[cherrypick for 1.13.8]

### PR DESCRIPTION
**Pull request by ispeakc0de**
It consists of following commits:

- bd278ef948aa098d0824cd9a71a2b9050ae1585f
- 00570a79dc557a71c13850e301b31e700ffc69b0
- 6c9b0b889f1a8b1610ab900b7a949eb095182ac0
- a5af71f24a9bd389e7ec4d628d37c7c60ba7d044
- 844a1453ee506201d8d53de09e7e49fd4f51af1e
- c808dfc90b052f0d0e6e93d156574796dba7d363
- 7a65033b80178ddc584e6799b189f921cb158357
- 83f182ef40feaa5a8ffcef5a4789d07e836877e2
- caa71e7a56f81e9c61522a17eecbfb8abdaee1b3
- e808bfcbc0620aa28c4e2ceba1aae221f38b3d7f

### CODE REVIEW

File: byoc/generic/k8-service-kill/k8-service-kill.chartserviceversion.yaml [hunk 1]
```diff
@@ -33,4 +33,4 @@ spec:
   icon:
     - url:
       mediatype: ""
-  chaosexpcrdlink: https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/generic/k8-service-kill/experiment.yaml
+  chaosexpcrdlink: https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/byoc/generic/k8-service-kill/experiment.yaml
```
Comment (uditgaurav) [inline byoc/generic/k8-service-kill/k8-service-kill.chartserviceversion.yaml:36]: for every CSV file the branch is master here can we update it.

File: byoc/generic/k8-service-kill/experiment.yaml [hunk 2]
```diff
@@ -44,10 +44,10 @@ spec:
     labels:
       name: k8-service-kill    
       app.kubernetes.io/part-of: litmus
-    image: "litmuschaos/chaostoolkit:latest"
+    image: "litmuschaos/py-runner:latest"
```
Comment (uditgaurav) [inline byoc/generic/k8-service-kill/experiment.yaml]:
```suggestion
    image: "litmuschaos/py-runner:1.13.8"
```

File: byoc/kube-aws/k8-aws-ec2-terminate/experiment.yaml [hunk 3]
```diff
@@ -41,10 +41,10 @@ spec:
         verbs :
           - "get"
           - "list"
-    image: "litmuschaos/chaostoolkit:latest"
+    image: "litmuschaos/py-runner:latest"
```
Comment (uditgaurav) [inline byoc/kube-aws/k8-aws-ec2-terminate/experiment.yaml]: 1.13.8

File: charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+          - "chaosexperiments"
+          - "chaosresults"
+        verbs:
+          - "create"
+          - "list"
+          - "get"
+          - "patch"
+          - "update"
+          - "delete"
+    image: "litmuschaos/go-runner:ci"
```
Comment (uditgaurav) [inline charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml]: 1.13.8

File: charts/aws-ssm/aws-ssm-chaos-by-tag/experiment.yaml [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+          - "chaosexperiments"
+          - "chaosresults"
+        verbs:
+          - "create"
+          - "list"
+          - "get"
+          - "patch"
+          - "update"
+          - "delete"
+    image: "litmuschaos/go-runner:ci"
```
Comment (uditgaurav) [inline charts/aws-ssm/aws-ssm-chaos-by-tag/experiment.yaml]: 1.13.8

File: charts/generic/pod-memory-hog-exec/experiment.yaml [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+          - "chaosresults"
+        verbs:
+          - "create"
+          - "list"
+          - "get"
+          - "patch"
+          - "update"
+          - "delete"
+          - "deletecollection"
+    image: "litmuschaos/go-runner:ci"
```
Comment (uditgaurav) [inline charts/generic/pod-memory-hog-exec/experiment.yaml]: 1.13.8

**Review by uditgaurav — APPROVED** [review]

**Review by oumkale — APPROVED** [review]

**Pull request merged**